PyTorch/diffusion/sd/app.py

```diff
@@ -0,0 +1,99 @@
+import torch
+import torch_directml
+import gradio as gr
+from diffusers import AutoPipelineForText2Image,  StableDiffusionPipeline, LMSDiscreteScheduler
+from PIL import Image
+import numpy as np
+ 
+def preprocess(image):
+    w, h = image.size
+    w, h = map(lambda x: x - x % 32, (w, h))  # resize to integer multiple of 32
+    image = image.resize((w, h), resample=PIL.Image.LANCZOS)
+    image = np.array(image).astype(np.float32) / 255.0
+    image = image[None].transpose(0, 3, 1, 2)
+    image = torch.from_numpy(image)
+    return 2.*image - 1.
+ 
+lms = LMSDiscreteScheduler(
+    beta_start=0.00085,
+    beta_end=0.012,
+    beta_schedule="scaled_linear"
+)
+ 
+device = torch_directml.device(torch_directml.default_device())
+ 
+block = gr.Blocks(css=".container { max-width: 800px; margin: auto; }")
+num_samples = 2
+ 
+def load_model(model_name):
+    return AutoPipelineForText2Image.from_pretrained(
+        model_name,
+        torch_dtype=torch.float16,
+        variant="fp16"
+        ).to(device)
```

Comment:
```suggestion
    ).to(device)
```
Consistent with other lines (like 21, 63, 79).